Network/AMQP/Internal.hs

```diff
@@ -604,6 +609,11 @@ channelReceiver chan = do
           modifyTVar' targetSet (\ts -> merge ts)
           writeTVar (unconfirmedSet chan) pending
 
+    handleCancel (ShortString deliveryTag) = do
```

Comment:
This parameter name is wrong. Consumers are identified by consumer tags, not delivery tags.